build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+/* eslint-disable unicorn/import-style */
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/promise-function-async */
+/* eslint-disable array-callback-return */
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+
+import * as fs from "fs";
+import { EOL as newline } from "os";
+import * as child_process from "child_process";
+import * as path from "path";
+import { Flags } from "@oclif/core";
+import {
+    copyrightFileHeaderHandlers,
+    npmPackageContentsHandlers,
+    dockerfilePackageHandler,
+    fluidCaseHandler,
+    lockfilesHandlers,
+    assertShortCodeHandler,
+    Handler,
+} from "@fluidframework/build-tools";
+import { BaseCommand } from "../../base";
+
+const readStdin: () => Promise<string | undefined> = () => {
+    return new Promise((resolve) => {
+        const stdin = process.openStdin();
+        stdin.setEncoding("utf-8");
+
+        let data = "";
+        stdin.on("data", (chunk) => {
+            data += chunk;
+        });
+
+        stdin.on("end", () => {
+            resolve(data);
+        });
+
+        if (stdin.isTTY) {
+            resolve("");
+        }
+    });
+};
+
+export class CheckPolicy extends BaseCommand<typeof CheckPolicy.flags> {
+    static description =
+        "Checks that the dependencies between Fluid Framework packages are properly layered.";
+
+    static flags = {
+        resolve: Flags.boolean({
+            description: `Resolve errors if possible`,
+            required: false,
+            char: "h",
+        }),
+        handler: Flags.string({
+            description: `Filter handler names by <regex>`,
+            required: false,
+            char: "h",
+        }),
+        path: Flags.string({
+            description: `Filter file paths by <regex>`,
+            required: false,
+            char: "p",
+        }),
+        stdin: Flags.boolean({
+            description: `Get file from stdin`,
+            required: false,
+            char: "s",
+        }),
+        ...BaseCommand.flags,
+    };
+
+    async run() {
+        const { flags } = await this.parse(CheckPolicy);
+        const exclusions: RegExp[] = require("../../data/exclusions.json").map(
+            (e: string) => new RegExp(e, "i"),
+        );
+        const handlerRegex =
+            typeof flags.handler === "string" ? new RegExp(flags.handler, "i") : /.?/;
+        const pathRegex = typeof flags.path === "string" ? new RegExp(flags.path, "i") : /.?/;
+        let pathToGitRoot = "";
+        let count = 0;
+        let processed = 0;
+
+        /**
+         * declared file handlers
+         */
+        const handlers: Handler[] = [
+            ...copyrightFileHeaderHandlers,
+            ...npmPackageContentsHandlers,
+            dockerfilePackageHandler,
+            fluidCaseHandler,
+            ...lockfilesHandlers,
+            assertShortCodeHandler,
+        ];
```

Comment:
You don't need the individual handlers here, so rather than export each of these handlers separately, create and export a new `Handler` array that you can use here. For example:

In the build-tools project:

```ts
export const allHandlers: Handler[] = [
            ...copyrightFileHeaderHandlers,
            ...npmPackageContentsHandlers,
            dockerfilePackageHandler,
            fluidCaseHandler,
            ...lockfilesHandlers,
            assertShortCodeHandler,
        ];
```

Then in build-cli, import `allHandlers.`